cfgov/jinja2/v1/_includes/molecules/call-to-action.html

```diff
@@ -0,0 +1,36 @@
+{# ==========================================================================
+
+   call-to-action.render()
+
+   ==========================================================================
+
+   Description:
+
+   Creates a call to action when given:
+
+   settings.text:        The text to include within slug markup.
+
+   settings.button:      Button options to create markup.
+
+   settings.button.href: A link to a page.
+                         Defaults to '/'.
+
+   settings.button.text: The text to display on button.
+
+   ========================================================================== #}
+
+{% macro render(settings={}, text='') %}
+<div class="m-call-to-action">
+    <p class="short-desc">
+        {{ settings.text }}
+    </p>
+    {% if caller %}
+        caller()
+    {% else %}
+        <a class="btn btn__full"
```

Comment:
fixed in https://github.com/cfpb/cfgov-refresh/commit/294a757262c9a0c51176affb0613b0facd93997b